test/browser_tests/conf.js

```diff
@@ -31,7 +31,7 @@ function _chooseSuite( params ) {
      won't launch several identical browsers performing the same tests. */
   let capabilities = defaultSuites.essential;
 
-  if ( envvars.HEADLESS_CHROME_BINARY ) {
+  if ( _useHeadlessChrome() ) {
```

Comment:
This change is so `headless` can be passed as a command-line argument? Is that correct?

Reply:
Yep, that is now supported in stable Chrome.